Integrations/integration-PaloAltoNetworks-pandb.yml

```diff
@@ -0,0 +1,464 @@
+            dbot_score = 2
+        elif (category == 'unknown'):
+            dbot_score = 0
+
+        return dbot_score
+
+
+    def integration_test():
+        """
+        test module
+        """
+        params = {
+            'type': 'op',
+            'cmd': '<show><system><info></info></system></show>',
+            'key': API_KEY
+        }
+
+        http_request(
+            URL,
+            'GET',
+            params=params
+        )
+
+        demisto.results('ok')
+
+
+    @logger
+    def get_url_category(url):
+        params = {
+            'action': 'show',
+            'type': 'op',
+            'key': API_KEY,
+            'cmd': '<test><url>' + url + '</url></test>'
+        }
+        result = http_request(
+            URL,
+            'POST',
+            params=params,
+        )
+
+        s = result['response']['result'].splitlines()[1]
+        return s.split(' ')[1]
+
+
+    def populate_url_filter_category_from_context(url_category_hr):
+        url_filter_category = demisto.dt(demisto.context(),
+                                         'URLFilter(val.Category === "{0}")'.format(url_category_hr['Category']))
+
+        if not url_filter_category:
+            url_filter_category = {
+                'Category': url_category_hr['Category'],
+                'URL': []
+            }
+
+        if type(url_filter_category) is dict:
+            url_filter_category = [url_filter_category]
+
+        url_filter_category[0]['URL'] += [url_category_hr['URL']]
+
+        return url_filter_category
+
+
+    def get_url_category_pandb():
+        """
+        URL Category as a standalone pandb command
+        """
+        url = demisto.args()['url']
+
+        category = get_url_category(url)
+
+        url_category_hr = {
+            'URL': url,
+            'Category': category
+        }
+
+        url_category_output = populate_url_filter_category_from_context(url_category_hr)
+
+        demisto.results({
+            'Type': entryTypes['note'],
+            'ContentsFormat': formats['json'],
+            'Contents': category,
+            'ReadableContentsFormat': formats['markdown'],
+            'HumanReadable': tableToMarkdown('PANDB [category](' + KB_ARTICLE + ')', url_category_hr, ['URL', 'Category'], removeNull=True),
+            'EntryContext': {
+                "URLFilter(val.Category === obj.Category)": url_category_output
+            }
+        })
+
+    def get_url_category_global():
+        """
+        URL Category as global URL category reputation
+        """
+        url = demisto.args()['url']
+
+        category = get_url_category(url)
+
+        url_category_hr = {
+            'URL': url,
+            'Category': category
+        }
+
+        url_category_output = populate_url_filter_category_from_context(url_category_hr)
+
+        dbot_scores = []
+        dbot_scores.append({
+            "Score"     : get_dbot_score(url, category),
```

Comment:
Please use `'` instead of `"` and make sure there are no white spaces between the key and the colon